docs/components/background.md

```diff
@@ -0,0 +1,27 @@
+---
+title: background
+type: components
+layout: docs
+parent_section: components
+source_code: src/components/background.js
+examples: []
+---
+
+The background component defines the background of a scene.
```

Comment:
Can we document the difference between `<a-sky>` and renderer clear color and benefits? Can also add to Best Practices page under Performance.

Reply:
Added